### DISCUSSION TITLE
Config.env file

### DISCUSSION THREAD

**Discussion by xmtscf** [phaticusthiccy/WhatsAsenaDuplicated#71] [Show and tell]
Could you provide an example of a config.env file, for those people who intend to use the userbot on their own server?

**Comment by phaticusthiccy** [comment]
Yeah, of course. 
I will add it to the wiki page soon.

**Comment by phaticusthiccy** [comment]
[Wiki](https://github.com/phaticusthiccy/WhatsAsenaDuplicated/wiki/config.env-Example)
Raw Link for Untouchable Text: https://github.com/phaticusthiccy/WhatsAsenaDuplicated/wiki/config.env-Example
